packages/react-core/src/components/Timestamp/Timestamp.tsx

```diff
@@ -0,0 +1,127 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Timestamp/timestamp';
+import { css } from '@patternfly/react-styles';
+import { Tooltip, TooltipProps } from '../Tooltip';
+
+export enum TimestampFormats {
+  full = 'full',
+  long = 'long',
+  medium = 'medium',
+  short = 'short'
+}
+
+export interface TimestampProps extends React.HTMLProps<HTMLSpanElement> {
+  /** Custom content rendered inside the timestamp, such as a relative time. Passing this prop
+   * in will override the default content and timeZoneSuffix prop.
```

Comment:
There is no content prop so I would reward this so it is more clear that it overrides the default display string.

Reply:
I think that is better.  I do not think it is necessary to mention the timeZoneSuffix prop.